### PR TITLE
Update equipped field in NftInfo

### PR DESCRIPTION
**Pull request by Szegoo**
Closes: #184

### CODE REVIEW

**Comment by Szegoo** [comment]
@mrshiposha PTAL.

**Review by ilionic — APPROVED** [review]
LGTM, thank you.
What about integration tests, no changes needed?

**Comment by Szegoo** [comment]
@ilionic I could probably update the `equipNft` function in the integration tests so that it checks for the resource and slot id.

**Comment by ilionic** [comment]
@Szegoo Yes, that would be great, thanks.

File: tests/src/equipNft.test.ts [hunk 1]
```diff
@@ -119,16 +119,34 @@ async function checkEquipStatus(
...
   expect(
-    itemNftData.equipped.isTrue,
+    itemNftData.equipped.isSome,
     `Error: item NFT should be ${expectedStatus}`
   ).to.be.equal(expectedStatus === "equipped");
+
+  if (
+    expectedStatus === "equipped" &&
+    expectedResourceId !== undefined &&
+    expectedSlotId !== undefined
```
Comment (ilionic) [inline tests/src/equipNft.test.ts:138]: minor thing, `!== undefined` is kinda redundant so a bit shorter and concise just to write as:
```
  if (expectedStatus === "equipped" && expectedResourceId && expectedSlotId)
```

Comment (Szegoo) [reply tests/src/equipNft.test.ts]: Yes, you are right, I updated this.

**Pull request merged**